zebra-state/src/constants.rs

```diff
@@ -31,3 +31,4 @@ lazy_static! {
     /// Regex that matches the RocksDB error when its lock file is already open.
     pub static ref LOCK_FILE_ERROR: Regex = Regex::new("(lock file).*(temporarily unavailable)|(in use)|(being used by another process)").expect("regex is valid");
 }
+
```

Comment:
Workflow triggers: Does PR #4237 avoid you having to make unrelated changes to Rust files to test this PR? Or do we need to make more workflow trigger fixes?

Reply:
We need to make more workflow trigger fixes:

I'm removing this change in the meanwhile. I thought of a solution for this triggers. I'll test that in another PR.

Fixed by: [fix: revert uneeded changes](https://github.com/ZcashFoundation/zebra/pull/4252/commits/f516ae5b636a84b20f70444847fe86ef8b6d551b)